joomla/joomla-cms
# [J5.2] Cassiopeia - Only add margin to system message container if it contains a joomla-alert

Pull Request for Issue #40639.

### Summary of Changes
Added a :has rule to control the margin of the system-message-container if it does not contain a <joomla-alert> tag
:has selector is supported by all modern browsers

<img width="1362" alt="grafik" src="https://github.com/user-attachments/assets/55380160-b299-4dce-836b-66a209aca4eb">

### Testing Instructions
login with wrong credentials in your cassiopeia frontend. 
A system message appears that the login data is wrong
Click the close button to hide the message


### Actual result BEFORE applying this Pull Request
After hiding a system alert, a margin of 1rem remains


### Expected result AFTER applying this Pull Request
After hiding a system alert, the margin is not present anymore

### Link to documentations



Please select:
- [x ] No documentation changes for docs.joomla.org needed

- [x ] No documentation changes for manual.joomla.org needed
